packages/lodestar-beacon-state-transition/src/fast/block/processOperations.ts

```diff
@@ -41,10 +42,12 @@ export function processOperations(
     [body.attesterSlashings, processAttesterSlashing],
     [body.attestations, processAttestation],
     [body.deposits, processDeposit],
-    [body.voluntaryExits, processVoluntaryExit],
+    [body.voluntaryExits, verifyVoluntaryExit],
```

Comment:
I wonder if we can modify this entire statement to
```ts
processAttesterSlashings(...)
processAttestations(...)
processDeposits(...)
```
And look for similar optimizations in the other opts

Reply:
the `initiateValidatorExit()` was called in `slashValidator.ts` also but we couldn't optimize the same way to this because it's required to process one by one there in order to use validator `withdrawableEpoch` correctly

other than that I don't see any similar improvement can be made so I keep other functions as is.